### ISSUE TITLE
Bug default_logging_config.ini in aca-py tag 0.2.1

### ISSUE THREAD

**Issue by SmithSamuelM** [hyperledger/aries-cloudagent-python#120]
When running aca-py  (version in tag 0.2.1) from the command line it gets an error because it can't find the default logging config. This is because when installed via pip from the PyPI repo, the default logging config file  at /,../aries_cloudagent/config/default_logging_config.ini' is not in the directory tree for the aca-py executable which is installed in /usr/local/bin/aca-py . This means there is no way to load the default config and unless the command line args supply a logging config it will always. fail. Running aca-py from a debugger from the repo will work because in this case aca-py is not in /usr/local/bin so the default init is in its local tree.  Typically for a command line python script to work it would need to look for default config files in ~/.myconfigdir or the setup.py needs to copy the file into some other well known location relative to the packages in site-packages.

Suggest instead embedding the default config as a resource in a module instead of a file.

The error
```bash
 aca-py --inbound-transport http 0.0.0.0 8020 --outbound-transport http --log-level debug --endpoint 'http://localhost:8020'
Traceback (most recent call last):
  File "/usr/local/bin/aca-py", line 25, in <module>
    main()
  File "/usr/local/lib/python3.7/site-packages/aries_cloudagent/__init__.py", line 56, in main
    LoggingConfigurator.configure(log_config, log_level)
  File "/usr/local/lib/python3.7/site-packages/aries_cloudagent/config/logging.py", line 30, in configure
    fileConfig(config_path, disable_existing_loggers=False)
  File "/usr/local/Cellar/python/3.7.4/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/config.py", line 71, in fileConfig
    formatters = _create_formatters(cp)
  File "/usr/local/Cellar/python/3.7.4/Frameworks/Python.framework/Versions/3.7/lib/python3.7/logging/config.py", line 104, in _create_formatters
    flist = cp["formatters"]["keys"]
  File "/usr/local/Cellar/python/3.7.4/Frameworks/Python.framework/Versions/3.7/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'formatters'


```

The code containing the error is here

```python
aclass LoggingConfigurator:
    """Utility class used to configure logging and print an informative start banner."""

    @classmethod
    def configure(cls, logging_config_path: str = None, log_level: str = None):
        """
        Configure logger.

        :param logging_config_path: str: (Default value = None) Optional path to
            custom logging config

        :param log_level: str: (Default value = None)
        """
        if logging_config_path is not None:
            config_path = logging_config_path
        else:
            config_path = path.join(
                path.dirname(path.abspath(__file__)), "default_logging_config.ini"
            )

        fileConfig(config_path, disable_existing_loggers=False)

        if log_level:
            log_level = log_level.upper()
            getLogger().setLevel(log_level)

```

To reproduce. 
$ pip3 install aries-cloudagent   
version 0.2.1

And call
$ aca-py 
but without providing  --log-config

Suggest adding a test pattern that tests aca-py in a real install not just in the repo.

**Comment by SmithSamuelM** [comment]
It doesn't matter what command line args are passed as long as the required ones are provided and --log-config is missing.

It always fails

**Comment by andrewwhitehead** [comment]
Resolved via PR #128
